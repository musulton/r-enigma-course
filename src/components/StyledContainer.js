import Styled from "styled-components";

const StyledContainer = Styled.div`
    margin: 30px;
    text-align: left;
`

export default StyledContainer
