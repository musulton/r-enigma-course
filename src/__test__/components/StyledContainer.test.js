import StyledContainer from "../../components/StyledContainer";
import {render} from "@testing-library/react";

describe("StyledContainer component", () => {
  test("should render correctly", () => {
    const {asFragment} = render(<StyledContainer />)

    expect(asFragment()).toMatchSnapshot()
  })
})