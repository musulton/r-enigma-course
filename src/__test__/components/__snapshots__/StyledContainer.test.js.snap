// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledContainer component should render correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-bgqQcB jPafiR"
  />
</DocumentFragment>
`;
